Resources/Prototypes/Nyanotrasen/Entities/Objects/Weapons/Melee/sword.yml

```diff
@@ -0,0 +1,26 @@
+- type: entity
+  name: wakizashi
+  parent: BaseItem
+  id: Wakizashi
+  description: Essentially a smaller katana. Fine plasteel craftwork.
```

Comment:
It's funny that the 2nd paragraph of Wikipedia describes them so:
`Wakizashi are not necessarily just a smaller version of the katana; they could be forged differently and have a different cross section.[[5]](https://en.wikipedia.org/wiki/Wakizashi#cite_note-5)`
Description could be better.

Reply:
I'll write something up.